elasticsearch-monitoring/values.yaml

```diff
@@ -0,0 +1,83 @@
+## Prometheus MatchLabel selectors
+prometheus:
+  labels:
+    app: "prometheus"
+    name: "k8s-monitor"
+    role: "alert-rules"
+
+## Interval for how often Prometheus scrapes the elasticsearch-exporter
+interval: "30s"
+
+## The following settings come from the elasticsearch-exporter chart
+elasticsearch-exporter:
+  ## number of exporter instances
+  ##
+  replicaCount: 1
+
+  ## restart policy for all containers
+  ##
+  restartPolicy: Always
+
+  image:
+    repository: justwatch/elasticsearch_exporter
+    tag: 1.0.2
+    pullPolicy: IfNotPresent
+
+  resources: {}
+    # requests:
+    #   cpu: 100m
+    #   memory: 128Mi
+    # limits:
+    #   cpu: 100m
+    #   memory: 128Mi
+
+  service:
+    type: ClusterIP
+    httpPort: 9108
+
+  es:
+    ## Address (host and port) of the Elasticsearch node we should connect to.
+    ## This could be a local node (localhost:9200, for instance), or the address
+    ## of a remote Elasticsearch server. When basic auth is needed,
+    ## specify as: <proto>://<user>:<password>@<host>:<port>. e.g., http://admin:pass@localhost:9200.
+    ##
+    uri: http://admin:pass@localhost:9200
+
+    ## If true, query stats for all nodes in the cluster, rather than just the
+    ## node we connect to.
+    ##
+    all: true
+
+    ## If true, query stats for all indices in the cluster.
+    ##
+    indices: true
+
+    ## Timeout for trying to get stats from Elasticsearch. (ex: 20s)
+    ##
+    timeout: 30s
```

Comment:
Your timeout and interval are the same. I would lower this timeout to 20 seconds and keep interval at 30 seconds. The exporter by default is even only 5 seconds https://github.com/justwatchcom/elasticsearch_exporter#configuration

Reply:
changed